src/Workspaces/Core/Portable/CodeActions/CodeActionWithOptions.cs

```diff
@@ -65,6 +66,14 @@ public abstract class CodeActionWithOptions : CodeAction
         /// <param name="cancellationToken">A cancellation token.</param>
         protected abstract Task<IEnumerable<CodeActionOperation>> ComputeOperationsAsync(object options, CancellationToken cancellationToken);
```

Comment:
we woudl likely want to make this method virtual once we have the public api for progress.  Forcing people to implement hte non-progress taking API doesn't make sense in that world.